package.json

```diff
@@ -13,7 +13,8 @@
   "scripts": {
     "start": "npm run dev",
     "dev": "gulp",
-    "build": "gulp build",
+    "build": "gulp build:v4",
+    "build:all": "gulp build:all",
```

Comment:
I'm not sold on these script names. 

Build is used by CI for releases so maybe we should have a `build:v3` and a `build:v4`. The one that gets kicked off depends on the tag you provide the CI job. Then `build` can inherently run both individual version builds.

For development, it's `npm start` or `npm run dev` for building and serving v4 so I was thinking we'd have a very similar command for building v3. Something like `npm run dev:v3` or `npm start -- v3`?